docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building the Kimai Docker
+
+The same docker file is used to build all the tagged images and is configured by combination of build arguments and 
+targets.
+
+## Targets
+
+The docker file has many staging targets but two functional builds are `prod` and `dev` which correspond to the 
+prod and development environments as outlined in the Kimai documentation.  The default is `prod`
+
+    docker build --target=prod .
+    docker build --target=dev .
+
+## Build Arguments
+
+### BASE
+
+  * `BASE=apache-debian`
```

Comment:
This indentations violates the markdownlint rule [MD007 - Unordered list indentation](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md007).